install.sh

```diff
@@ -106,6 +110,26 @@ pre_flight_checks() {
   PRODUCTS_ARRAY=($(echo $PRODUCTS | sed 's/\[//g' | sed 's/\]//g' | sed 's/,/ /g' | sed 's/"//g'))
   REGION=$(get_variable 'region' "${CONFIG_ABS_PATH}")
 
+  if [ "${LICENSE_TEST_FLAG}" == "" ]; then
+    for PRODUCT in ${PRODUCTS_ARRAY[@]}; do
+      log "Checking ${PRODUCT} license"
+      LICENSE_ENV_VAR=${PRODUCT}'_license'
+      LICENSE_TEXT=$(get_variable ${LICENSE_ENV_VAR} "${CONFIG_ABS_PATH}")
+      if [ -z "$LICENSE_TEXT" ]; then
+        log "License is undefined or malformed. Please check '${LICENSE_ENV_VAR}' value in '${CONFIG_ABS_PATH}'" "ERROR"
+        log "It is possible that '${LICENSE_ENV_VAR}' is defined but it is a multi line string" "ERROR"
+        log "If that's the case remove all new lines to convert it to a one-liner" "ERROR"
+        exit 1
+      fi
+      if [ "${LICENSE_TEXT}" == ${PRODUCT}-license ]; then
```

Comment:
The placeholder value `${PRODUCT}-license` comes from [config file in DCAPT repo](https://github.com/atlassian/dc-app-performance-toolkit/blob/master/app/util/k8s/dcapt-small.tfvars#L78C23-L78C23). 
If they change the placeholder value, which is unlikely, we'll need to change here as well.